dask/array/core.py

```diff
@@ -610,13 +610,27 @@ def map_blocks(
             'chunk-shape': (100,),
             'dtype': dtype('float64')}}
 
-    For each argument and keyword arguments that are dask arrays (the positions
-    of which are the first index), you will receive the shape of the full
-    array, the number of chunks of the full array in each dimension, the chunk
-    location (for example the fourth chunk over in the first dimension), and
-    the array location (for example the slice corresponding to ``40:50``). The
-    same information is provided for the output, with the key ``None``, plus
-    the shape and dtype that should be returned.
+    The keys to the ``block_info`` dictionary indicate which is the input and
```

Comment:
Suggestion for this first sentence: During computation, ``block_info`` contains information about each input and output chunk (and dask array) relevant to each call of ``func``:

Reply:
I've added this sentence (up a bit higher in the text, where we first introduce `block_info` and `block_id`).